data/json/recipes/weapon/ranged.json

```diff
@@ -17,7 +17,68 @@
     "components": [
       [ ["nailgun", 1] ],
       [ ["pipe", 1] ],
-      [ ["2x4", 2], [ "stick", 3 ] ]
+      [
```

Comment:
I see, I could have sworn the preferred styling used to be one-line only if there is no choice of ingredient. How outdated is that styling method?